routers/bird/README.md

```diff
@@ -191,23 +198,26 @@ protocol bgp neighbor_v4_1 {
   neighbor 10.99.182.128 as 65530;
   password "somepassword";
 }
-```
-
-Check that the config looks correct, then restart bird:
-
-```bash
-systemctl restart bird
++ '[' 0 == 0 ']'
++ echo
++ cat
++ supervisord -c /opt/bgp/routers/bird/supervisord.conf
+2020-04-09 13:44:27,263 WARN For [program:bird], AUTO logging used for stderr_logfile without rollover, set maxbytes > 0 to avoid filling up filesystem unintentionally
+2020-04-09 13:44:27,263 INFO Set uid to user 0 succeeded
+2020-04-09 13:44:27,265 INFO supervisord started with pid 12
+2020-04-09 13:44:28,270 INFO spawned: 'bird' with pid 16
+2020-04-09 13:44:29,274 INFO success: bird entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
 ```
 
 And verify:
 
 ```bash
-birdc
+docker exec -it $(docker ps | awk '$2 == "local/bird:latest" {print $1}') birdc
```

Comment:
I would prefer using built-in filtering -> https://docs.docker.com/engine/reference/commandline/ps/#filtering